{"name": "Принять заказ. Попытка принять заказ с некорректным id курьера возвращает корректное сообщение об ошибке", "status": "passed", "description": "тест ручки \"Принять заказ\"", "start": 1724853582851, "stop": 1724853583073, "uuid": "d7c2a5e6-40c8-4851-8ccb-505b650768ae", "historyId": "dc56c8611b5ea8558e0798144e5a2c53", "testCaseId": "dc56c8611b5ea8558e0798144e5a2c53", "fullName": "tests.TestAcceptOrder#test_try_accept_order_with_non_existing_courier_id", "labels": [{"name": "suite", "value": "tests"}, {"name": "subSuite", "value": "TestAcceptOrder"}, {"name": "host", "value": "FanatovPC"}, {"name": "thread", "value": "31284-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests"}]}